src/DataCollector/LogsCollector.php

```diff
@@ -123,11 +114,17 @@ public function getLogs($file)
             }
         }
 
-        $log = array_reverse($log);
-
         return $log;
     }
 
+    /**
+     * @return array
+     */
+    public function getMessages()
+    {
+        return array_reverse(parent::getMessages());
+    }
+
```

Comment:
@erikn69 am I right that this function isn't called anywhere?

Reply:
This is an important part to show the most recent logs first, Means, descending, according to log timestamp.

Reply:
inheritance from parent MessagesCollector